src/holofuel/components/SideMenu/SideMenu.js

```diff
@@ -106,10 +106,10 @@ export function SideMenu ({
       <footer styleName='footer'>
         <ul styleName='footer-list'>
           <li>
-            <Link to='/tos' styleName='footer-link'>View Disclaimer</Link>
+            <Link to='#/' styleName='footer-link'>View Disclaimer</Link>
```

Comment:
does this mean we can remove the `TOS` page entirely?

Reply:
The TOS has been removed from the routing page, so I changed this link to a placeholder to avoid erroring out.  Has the design changed to remove this entirely? I would like to defer to the designs to answer this question.

Reply:
Ok. So I checked and it looks like TOS is still apart of the sidebar menu, but disclaimer is not.  I made the updates accordingly. :+1: